staff/lab/ocf-tv

```diff
@@ -43,15 +43,23 @@ def wait_for_port(host, port, timeout=5, ssh_proc=None):
 
 
 def volume(args):
-    call(('ssh', args.host,
-          'export PULSE_SERVER=localhost; pactl set-sink-volume {} {}%'.format(args.sink, args.amount)))
+    cmd = 'export PULSE_SERVER=localhost; '
+
+    if args.amount is not None:
+        cmd += 'pactl set-sink-volume 0 {}%; '.format(args.amount)
+
+    # thanks SO for awk tricks
+    cmd += "pactl list sinks | awk '/Sink #0/{nr[NR+9]}; NR in nr' | cut -d' ' -f 6;"
```

Comment:
The only part of this that I found on SO was using nr[NR+x] to get some line number after the match. Beyond that adding numerous variations of what should work in awk (`{print $6}`) doesn't seem to work, and all the examples I could find suggest to just farm it out to cut.